src/item.h

```diff
@@ -1214,6 +1214,7 @@ class item : public JsonSerializer, public JsonDeserializer
 
  int quiver_store_arrow(item &arrow);
  int max_charges_from_flag(std::string flagName);
+ int get_gun_ups_drain(); 
```

Comment:
The function should be const (as in `int get_gun_usp_drain() const;`) as it's supposed to not change the item.